docs/source/features_conceptual/content_word_accommodation.rst

```diff
@@ -0,0 +1,55 @@
+
+**Prepositions**
+  about, after, against, at, before, by, down, for, from, in, into, near, of, off, on, out, over, than, to, until, up, with, without
+
+**Discourse Particles**
+  ah, hi, huh, like, mm-hmm, oh, okay, right, uh, uh-huh, um, well, yeah, yup
+
+**Adverbs and Negatives**
+  just, no, not, really, too, very
+
+Implementation Notes/Caveats 
+*****************************
+Note that the first utterance in a conversation cannot have a mimicry score, as there is no "previous utterance" to associate it with. In this case, we assign a value of 0 to this utterance.
+
+Interpreting the Feature 
+*************************
+This feature generates a (normalized) shared word count for each utterance in a conversation, with lower scores (close to 0) representing utterances that discuss more "different" content compared to the previous utterance; in other words, there is a lack of content word mimicry. Higher scores represent utterances that discuss the "same" content compared to the previous utterance.
+
+It's important to note that this score doesn't measure the overall level of mimicry over the course of the conversation. As an utterance-level feature, it computes the content word mimicry only between the focal utterance and the previous one. It's also important to note that, because the feature is lexical, it doesn't measure "mimicry" in the sense of having a simialr opinion --- only in the sense of having a large number of shared words. For example, "I loved the movie last night" and "I hated the movie last night" express opposing sentiments, but share most of their content words ("movie", "last", "night"). Additionally, homonyms may create false positives; "I was present in class" and "I got you a present" share the content word "present," but it takes on a different meaning in each case. Other measures of mimicry relying on transformer-based models (e.g., the Mimicry (BERT) features) can help to mitigate this issue.
+
+Related Features 
+*****************
+Other mimicry-related features include Function Word Accommodation, Mimicry (BERT), and Moving Mimicry. Function Word Accommodation is a lexical feature that is the complement of this one; it measures the number of function words shared between successive utterances. The latter two use more advanced, transformer-based models to compute similarity between utterances. Mimicry (BERT) uses the cosine similarity between sBERT embeddings to measure mimicry between a given utterance and the previous one. Moving Mimicry is similar to Mimicry (BERT) in that it uses sBERT embeddings to compute similarity, but differs in that it helps reason towards the overall flow of mimicry throughout a conversation, rather than at one instantaneous point.
```

Comment:
Can you elaborate more on "helps reason towards the overall flow of mimicry throughout a conversation, rather than at one instantaneous point"? I'm not quite sure what this means.